### ISSUE TITLE
During playback, when opening the movie information window, auto-scrolling of the text in the field "Critics consensus" does not work and it is impossible to read the text to the end.

### ISSUE THREAD

**Issue by alexeichikad** [jurialmunkey/skin.arctic.horizon.2#960]
### Skin section

Fullscreen Video OSD

### Current Behavior

During playback, when opening the movie information window, auto-scrolling of the text in the field
"Critics consensus" does not work and it is impossible to read the text to the end.

### Expected Behavior

During playback, when opening the movie information window, auto-scrolling of the text in the field
"Critics consensus" does not work and it is impossible to read the text to the end.

### Steps To Reproduce

During playback, when opening the movie information window, auto-scrolling of the text in the field
"Critics consensus" does not work and it is impossible to read the text to the end.

### Screenshots and Additional Info

![screenshot00000](https://github.com/jurialmunkey/skin.arctic.horizon.2/assets/127650429/19c81564-e76c-4d7e-a790-735fafc00f5e)

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Yeah I know about this one. It's an unfortunate compromise that needs to be made due to a bug in the skinning engine with textbox height calculation. See this comment for more details: https://github.com/jurialmunkey/skin.arctic.horizon.2/issues/865#issuecomment-1465082632

Kodi miscalculates auto-height of textbox when the font has linespacing and always makes it too short for the text. If autoscroll is enabled then it will *always* scroll too far up and overshoot because it thinks the textbox is slightly too short. 

That means that if there is only one or two lines of text you end up not being able to see them because it scrolls them completely offscreen even if there is enough space.

There's no solution other than it being fixed in Kodi.

**Comment by jurialmunkey** [comment]
Actually, sorry I see you're talking about critics consensus not awards. 

Critics consensus has auto-scroll enabled in the skin, so it's not the exact same issue as in the other thread but still has the same underlying cause due to Kodi miscalculating auto-height of a textboxes with linespacing. 

Until it is fixed in Kodi, not much skins can do about it unfortunately.

**Comment by alexeichikad** [comment]
Thanks for the answer.  Is it possible to add an option to disable the
display of Awards or Critics Consensus information
(1080i/Includes_OSD_CastInfo.xml)?  For example, if in the skin settings
for Ratings there is no switch to show Awards or Rottentomatoes Critiics
for a movies, then not show this data also in OSD?

ср, 19 июл. 2023 г., 04:50 jurialmunkey ***@***.***>:
